eng/pipelines/maintenance-packages.yml

```diff
@@ -96,7 +96,43 @@ stages:
               $(_InternalBuildArgs)
             displayName: $(_BuildJobDisplayName)
 
+- stage: pack
+  displayName: Pack
+  dependsOn: [] # To run in parallel
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      artifacts:
+        publish:
+          artifacts: false
+          logs: true
+          manifests: true
+      enableMicrobuild: true
+      enablePublishUsingPipelines: false
+      publishAssetsImmediately: false
+      enablePublishTestResults: false
+      workspace:
+        clean: all
+      jobs:
+      - ${{ if eq(variables._RunAsPublic, True) }}:
+        - job: linux
+          timeoutInMinutes: 60
+          container: LinuxContainer
+          pool:
+            vmImage: ubuntu-latest
+          preSteps:
+          - checkout: self
+            clean: true
+          steps:
+          - script: eng/common/build.sh
+              --configuration Release
+              --restore
+              --build
+              --pack
+              /p:IsSourceProjectPackable=true
+            displayName: $(_BuildJobDisplayName)
```

Comment:
```suggestion
            displayName: Build
```

Reply:
I used "Restore & Pack", let me see if that works and how it shows up.